src/modules/commentpop.js

```diff
@@ -176,7 +177,7 @@ const openPopup = async (countryDetails) => {
       displayComment(comment);
     });
   } catch (error) {
-    return error.message;
+    console.log('Error:', error);
```

Comment:
- [ ] This issue as pointed out by the previous reviewer is not yet resolved :x: . Please, remove all `console.log()` statements from your app.